systems/data-plan/package/pkg/db/package_repo.go

```diff
@@ -18,7 +18,7 @@ import (
 )
 
 type PackageRepo interface {
-	Add(_package *Package) error
+	Add(packageData *Package, packageRate *PackageRate) error
```

Comment:
Maybe we can just name it `dataPackage`. I'm not sure why we're adding a data package and then returning the rate instead of the package itself.

Reply:
Here `packageRate` is an argument not a return type.